### PR TITLE
Use consistent format for verbs

### PR DESCRIPTION
**Pull request by ianstanton**
Define verbs in a format that is consistent with the rest of the file.
Signed-off-by: Ian Stanton <ian@coredb.io>

### CODE REVIEW

File: yaml/deployment.yaml [hunk 1]
```diff
@@ -21,11 +21,7 @@ metadata:
 rules:
   - apiGroups: ["kube.rs"]
     resources: ["documents", "documents/status"]
-    verbs:
-    - get
-    - watch
-    - list
-    - patch
+    verbs:["get", "list", "watch", "patch"]
```
Comment (clux) [inline yaml/deployment.yaml:24]:
```suggestion
    verbs: ["get", "list", "watch", "patch"]
```
minor lint

**Review by clux — APPROVED** [review]
Thanks!

**Comment by clux** [comment]
Oh, actually, this file is generated from the helm chart now, so it needs to be done in the template.

**Comment by ianstanton** [comment]
> Oh, actually, this file is generated from the helm chart now, so it needs to be done in the template.

Got it, will update

**Pull request merged**